epimodel/exports/epidemics_org.py

```diff
@@ -432,22 +472,20 @@ def process_export(args) -> None:
     )
 
     for code in export_regions:
-        reg = args.rds[code]
+        reg: Region = args.rds[code]
         m49 = int(reg["M49Code"])
         iso3 = reg["CountryCodeISOa3"]
 
-        ##### TODO: go over simulation groups (mitigations), for each get stats and plots
-        for group in set(simulation_specs.Group):
-            sim_ids = list(simulation_specs[simulation_specs.Group == group].index)
-            stats = batch.generate_sim_stats(cummulative_active_df, reg, sim_ids)
-            # TODO: do something with the stats
-            print(reg, group, stats)
-            # TODO: plot Active as a curve
```

Comment:
@gavento this is also where I am unsure - does it the current code address or not at all?

Reply:
Composing Infected and Recovered to Active does not work on frontend. We need to expose the timeseries of `commulative_active_df` instead.

Reply:
In my opinion there are two ways to move forward with this. 
1) Either keep sending the `deltas` to the frontend but use Thomas method this method to compute the correct initial offset which can just replace `initial_df`. 
2) Or just remove the old traces and use only `cummulative_active_df.

Is this answer to the right question?

Reply:
Thanks, that's definitely helpful. Let me pull out a version with cummulative_active_df

Reply:
So would this commit solve the problem: https://github.com/epidemics/epimodel/pull/34/commits/ea4bd32608d7226d10b33aceddaa08e3d6b2f9a5? 

I basically put Infected, Active and Recovered in the traces from the cummulative DF.